### ISSUE TITLE
Do you have any plan for xstate v5?

### ISSUE THREAD

**Issue by CreadDiscans** [SimeonC/storybook-xstate-addon#57]
xstate v5 has many different from v4.

So the package need migration works.

https://stately.ai/docs/migration

Do you have any plan about this?

**Comment by SimeonC** [comment]
Yup, I will update this when we migrate at work. Open to PRs to migrate earlier as we're in the middle of a webpack to Vite upgrade and are waiting for the VSCode extension to support v5 so it may be a few months before I get time to look at this.

**Comment by HighLiuk** [comment]
@SimeonC FYI the VSCode extension to support v5 has finally been released!
